style.css

```diff
@@ -50,7 +50,7 @@ body {
 .menue-popup {
```

Comment:
- [ ] Kindly increase the size of the hamburger
- Your Design
<img width="505" alt="Screenshot 2023-05-23 at 13 18 48" src="https://github.com/alizaamir123/mobile-version/assets/81444338/6442ae3c-218f-465a-b5c9-e9036a4ee6e0">
- Figma Design
<img width="961" alt="Screenshot 2023-05-23 at 13 24 50" src="https://github.com/alizaamir123/mobile-version/assets/81444338/5ca9632b-e3c9-483c-8ed1-277ab0ea93db">